plugins/ctr.yaml

```
@@ -0,0 +1,40 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: ctr
+spec:
+  version: v0.0.1
+  homepage: https://github.com/cfanbo/kubectr
+  shortDescription: list all containers in a pod quickly
+  description: |
+    kubectr is a utility that displays all containers in a pod, supporting 
```

```suggestion
    ctr is a utility that displays all containers in a pod, supporting 
```
When distributed as a plugin, `kubectr` is not the name of the tool.
I recommend accommodating this in the source code as well when args[0]=kubectl*